examples/src/stt.ts

```diff
@@ -25,7 +25,7 @@ export default defineAgent({
       const recvTask = async () => {
         for await (const event of sttStream) {
           if (event.type === stt.SpeechEventType.FINAL_TRANSCRIPT) {
-            console.log(event.alternatives[0].text);
+            console.log(event.alternatives[0]!.text);
```

Comment:
is it safe to assume that final transcript will have `event.alternatives` populated under all circumstances with at least one element?

Reply:
yes because we check for it inside the STT class. this is a one-or-more array

Reply:
we could define it as a one-or-more array on the typescript side then with `alternatives: [elementType, ...elementType[]`

Reply:
well actually not quite. it's an empty array in case of `{START,END}_OF_SPEECH` because it's unused and it's easier to make it empty than have some sort of complicated typescript type depending on enum value for this. but for final transcript it is guaranteed. maybe i can make alternatives `|undefined` and then the `!` would be on it instead for better readability

Reply:
sounds good!

Reply:
already pushed!